frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -99,16 +103,10 @@ export const detectOCS: FeatureDetector = async (dispatch) => {
 
 export const detectOCSSupportedFeatures: FeatureDetector = async (dispatch) => {
   try {
-    const subscription = await fetchK8s<K8sResourceKind>(
-      SubscriptionModel,
-      'ocs-subscription',
-      CEPH_STORAGE_NAMESPACE,
-    );
-    const ocsCSV = await fetchK8s(
-      ClusterServiceVersionModel,
-      subscription?.status?.currentCSV,
-      CEPH_STORAGE_NAMESPACE,
-    );
+    const csvList = await fetchK8s(ClusterServiceVersionModel, '', CEPH_STORAGE_NAMESPACE);
+    // eslint-disable-next-line dot-notation
+    const csvItems = csvList['items'] as ClusterServiceVersionKind[];
```

Comment:
```suggestion
    const csvItems = csvList?.items as ClusterServiceVersionKind[];
```

Reply:
`csvItems` will not be an array by type, so using dot-notation will throw compile-time error.